### DISCUSSION TITLE
Is it come with the price or free?

### DISCUSSION THREAD

**Discussion by danangprass** [pockethost/pockethost#209] [Q&A] [answered]
hai,

i just wondering is it come with the price or free?
since i don't see any price page at all..

hopefully someone can give me some insight.
thank you

**Comment by iampopal** [comment]
I am also wondering is it free or how much they are about to cost permonth and how long is it free?

**Comment by benallfree** [comment, marked as answer]
Hey guys, yes, PocketHost is completely free and we have no plans to begin charging. If we do introduce paid tiers, there will still be a free tier. I hope that helps!

**Reply by nicosql** [reply]
@benallfree - it doesn't help lol please explain WHY you would do such a kind thing?   

It's odd but as much as I love love love love love this idea and LOVE pocketbase, not being able to pay money for some kind of "pro" tier or something gives me pause for using it beyond anything but testing lol

I jest a little but I am also selfishly implying you really should consider perhaps a pro tier that comes with options like (for example) Litestream integration or even offer that service yourself so backups can be maintained etc..
Some other great additions for pro:
- private vm / higher performance
- option to upload a custom pocketbase build (the author encourages use as a framework as I'm sure you know)
- magiclinks auth option


overall, great product and quite wise timing I would argue, I think pocketbase is going to continue to explode in popularity.  Could be the flagship framework GOLANG has been dying for.

Thank you !!!

edit:  I will leave my shame here for all to see but I see you are already on top of most of these things lol I just didn't read thoroughly.
